solc/CommandLineInterface.cpp

```diff
@@ -668,7 +668,7 @@ void CommandLineInterface::handleCombinedJSON()
 		if (requests.count("srcmap-runtime"))
 		{
 			auto map = m_compiler->runtimeSourceMapping(contractName);
-			contractData["srcmap"] = map ? *map : "";
+			contractData["srcmap-runtime"] = map ? *map : "";
```

Comment:
Yes it should, but it is using the `-` style in all other fields.

We can unify the styles at the point where we transition to the input being json-encoded, then we do not break backwards-compatibility.